app/retail/templates/shared/nav.html

```diff
@@ -87,18 +87,10 @@
       {% else %}
       <a class="nav-link{% if active == 'tools' %} selected{% endif %} interior" href="{% url "tools" %}">{% trans "Tools" %}</a>
       {% endif %}
-      {% if request.path != '/' %}
-        <a href="{% url "new_funding" %}" class="nav-link interior button--fund">
-          <i class="fab fa-ethereum mr-1" role="aria-hidden"></i>
-          {% trans "FUND ISSUE" %}
-        </a>
-      {% else %}
-        <a href="{% url "new_funding" %}" class="btn btn-gc-blue interior button--fund">
-          <i class="fab fa-ethereum mr-1" role="aria-hidden"></i>
-          {% trans "FUND ISSUE" %}
-        </a>
-      {% endif %}
-
+      <a href="{% url "new_funding" %}" class={% if request.path != '/' %}"nav-link"{% else %}"btn btn-gc-blue interior button--fund"{% endif %}>
```

Comment:
Doesn't `button--fund` need to be applied for both scenarios?

Reply:
Yes, fixed! 😄